src/Hero/Hero.stories.tsx

```diff
@@ -0,0 +1,116 @@
+// REACT
+import * as React from 'react';
+// STORYBOOK
+import { storiesOf } from '@storybook/react';
+import styled from 'styled-components';
+import { text, color } from '@storybook/addon-knobs';
+// ANCHOR
+import { Typography } from '..';
+import { colors } from '../theme';
+import * as README from './README.md';
+import SomeImage from './temporary/some_image.svg';
```

Comment:
Let's create this inside the source code for this story. If you need help, LMK.

Reply:
You mean defining the svg component inside the story that uses it? Seems like that'd add a lot of noise to this file.